python/dgl/contrib/dis_sampling/bench_receiver.py

```diff
@@ -0,0 +1,26 @@
+# Benchmark receiver
```

Comment:
Remove this file from the main package. You could put it on a gist or upload to https://github.com/jermainewang/dgl-workspace . Please ping me if you don't have the access.